core/cosmosdb/cache-invalidator/build.gradle

```diff
@@ -22,7 +22,8 @@ apply plugin: 'maven'
 apply plugin: 'org.scoverage'
 
 ext.dockerImageName = 'cache-invalidator-cosmosdb'
-apply from: "${project.rootProject.projectDir.absolutePath}/gradle/docker.gradle"
+apply from: "../../gradle/docker.gradle"
```

Comment:
It's `../../../gradle/docker.gradle` and remove the comment.

Reply:
@dubee It's now fixed.